homeassistant/components/zwave/node_entity.py

```diff
@@ -75,6 +75,7 @@ def __init__(self, node, network):
         super().__init__()
         from openzwave.network import ZWaveNetwork
         from pydispatch import dispatcher
+        self.component = None
```

Comment:
This is not necessary. Entities already have access to the platform that contains them at `self.platform`.

Reply:
Maybe the "remove, clear_id, add back" functionality should be added to the Entity class then? WDYT?

Reply:
I don't think that it's a very common use case to do this. Better keep it in Z-Wave, we can always make it a generic function once we see more places need it.

Reply:
Ack

Reply:
This can be removed since it's no longer being used?

Reply:
Done